src/designer/elsa-workflows-studio/src/components/screens/workflow-blueprint-viewer/elsa-workflow-blueprint-viewer-screen/elsa-workflow-blueprint-viewer-screen.tsx

```diff
@@ -90,6 +90,7 @@ export class ElsaWorkflowBlueprintViewerScreen {
   async loadActivityDescriptors() {
     const client = await createElsaClient(this.serverUrl);
     state.activityDescriptors = await client.activitiesApi.list();
+    console.log(state.activityDescriptors);
```

Comment:
Not sure if you added it or me or someone else, but perhaps we should remove this?